action/protocol/rewarding/reward.go

```diff
@@ -117,7 +122,7 @@ func (p *Protocol) GrantBlockReward(
 	rewardLog := rewardingpb.RewardLog{
 		Type:   rewardingpb.RewardLog_BLOCK_REWARD,
 		Addr:   rewardAddrStr,
-		Amount: a.blockReward.String(),
+		Amount: totalReward.String(),
```

Comment:
should we define another type like `RewardLog_PRIORITY_FEE`? so we don't mingle this with block reward